Elastiflix/java-favorite-otel-auto/Dockerfile

```diff
@@ -15,8 +15,14 @@ RUN mvn clean install
 
 USER root
 RUN apt-get update && apt-get install -y zip curl
+
+ARG OTEL_AGENT_VERSION=1.28.0
 RUN mkdir /otel
-RUN curl -L -o /otel/opentelemetry-javaagent.jar https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.28.0/opentelemetry-javaagent.jar
+RUN curl -L -o /otel/opentelemetry-javaagent.jar https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v${OTEL_AGENT_VERSION}/opentelemetry-javaagent.jar
+
+# For now use only the latest snapshot
+RUN mkdir /elastic
```

Comment:
[for reviewer] using snapshot is only temporary until we have a first release of the agent.